### PR TITLE
feat: PFNano for Run2022 re-reco data and Run 3 Summer22(EE) MC

### PR DESCRIPTION
**Pull request by demuller**
Changes compared to `12_4_8` branch:
- added/modified python config files for producing PFNano samples for re-reco Run2022 data (runs ABCD) and for Run 3 Summer22(EE) MC samples
- adjusted example yml files
-  adjusted README.md
-  modified `eventcontent` and `datatier` for data from `NANOAODSIM` to `NANOAOD`
-  moved `fakeNameforCrab` modifier from `pfnano_cff.py` to `cmsDriver` commands

The XPOG recommendations for re-reco Run2022 data and Run 3 Summer2022(EE) MC can be found [here](https://gitlab.cern.ch/cms-nanoAOD/nanoaod-doc/-/wikis/Releases/NanoAODv11)

Notes:
- a global tag recommendation for re-reco Run2022E exists and is different compared to ABCD, but could not be tested as the MINIAOD samples have not been produced yet, will be addressed in a follow-up PR
-  re-reco Run2022FG are currently WIP, will be addressed in a follow-up PR

### CODE REVIEW

File: test/card_example_dataABCD.yml [hunk 1]
```diff
@@ -3,17 +3,17 @@ campaign:
...
   outLFNDirBase: /store/user/anstein/PFNano # Change username and path
   voGroup: dcms # or leave empty
 
   # Campaign specific
-  tag_extension: BTV_Run3_2022_Comm_v1 # Will get appended after the current tag
+  tag_extension: BTV_Run3_2022_Comm_v2 # Will get appended after the current tag
   tag_mod: # Will modify name in-place for MC eg. "PFNanoAODv1" will replace MiniAODv2 -> PFNanoAODv1
   # If others shall be able to access dataset via DAS (important when collaborating for commissioning!)
   publication: True
-  config: nano_data_2022_NANO.py
```
Comment (AnnikaStein) [inline test/card_example_dataABCD.yml:16]: We could then delete the PromptReco config (nano_data_2022_NANO.py) from this branch, right? As it's not run anymore in any card.

Comment (AnnikaStein) [reply test/card_example_dataABCD.yml:16]: Even if we take into account that for 2022FG there will be PromptReco in nano v11 and not yet re-reco, we will anyway need another config with new GT for that. So the old nano_data_2022_NANO.py will be outdated in any case and can be removed.

File: python/pfnano_cff.py [hunk 2]
```diff
@@ -68,46 +68,46 @@ def PFnano_customizeMC_noInputs(process):
 def PFnano_customizeData(process):
     addPFCands(process, False)
     add_BTV(process, False, keepInputs=['DeepCSV','DDX'])
-    process.NANOAODSIMoutput.fakeNameForCrab = cms.untracked.bool(True)  # needed for crab publication
+    process.NANOAODoutput.fakeNameForCrab = cms.untracked.bool(True)  # needed for crab publication
```
Comment (AnnikaStein) [inline python/pfnano_cff.py:71]: In PR https://github.com/cms-jet/PFNano/pull/46 there was the proposal to not put these modifiers into the `cff` but instead give them with each `cmsDriver` command. I think both are fine, however we could unify the config generation and synchronize across branches as much as possible by moving to the `--customize_commands` variant here as well and denote that in the readme as well.

Comment (AnnikaStein) [reply python/pfnano_cff.py:71]: (Same comment for all entries in `pfnano_cff.py`, not only for Data actually)

**Review by AnnikaStein — COMMENTED** [review]
Thanks @demuller for the updated configs towards the new campaign!
Only left a few comments related to synchronizing across branches and cleaning up unused configs, but none of them are mandatory for merging.
Best,
A.

File: test/card_example_dataABCD.yml (hunk 1 above)
Comment (demuller) [reply test/card_example_dataABCD.yml:16]: My bad, I forgot to delete the old config, and forgot to replace it in the yml file with the re-reco one!

**Comment by AnnikaStein** [comment]
Thanks for the prompt updates, looks good now!

**Pull request merged**